src/stores/user.js

```diff
@@ -60,21 +66,40 @@ export const useUserStore = defineStore('user', {
         })
     },
 
+    async googleSignInWithEmailAndPassword() {
+      this._isLoading = true
+
+      await signInWithEmailAndPassword(auth, "test@test.com", "12345678")
```

Comment:
Eventually, I would like users to be able to make us of signInWithEmailAndPassword() so may as well have it here. I'll suggest though that we somehow export the <username> & <password> into the "/.env.test" file.